assets/css/main.css

```diff
@@ -31,17 +29,31 @@ html,body {
     color: white;
 }
 
-h1 {
+body > div {
+    padding: 1rem 0;
```

Comment:
The main body panels feel really tight. Especially on mobile, it's making it feel like the page is too cramped to read. I'd like to bump this value up to

```css
padding: 3rem 0;
```

to add a little extra space for readability.